src/test/librados/TestCase.cc

```diff
@@ -13,7 +14,8 @@ rados_t RadosTestNS::s_cluster = NULL;
 
 void RadosTestNS::SetUpTestCase()
 {
-  pool_name = get_temp_pool_name();
+  auto PoolPrefix = fmt::format("{}_", ::testing::UnitTest::GetInstance()->current_test_case()->name());
```

Comment:
please following our naming convention. `PoolPrefix` should be `pool_prefix`.

Reply:
Done